index.html

```diff
@@ -40,6 +40,129 @@ <h5>LET’S CONNECT</h5>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case3">
+          <div class="content">
+            <h4>Tonic</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case4">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+      </section>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><img src="imgs/Github.png" alt="Github Icon" /></li>
+          <li><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></li>
+          <li><img src="imgs/hands.png" alt="hands" /></li>
+          <li><img src="imgs/Twitter.png" alt="Twitter Icon" /></li>
+          <li><img src="imgs/M.png" alt="M icon" /></li> 
+        </ul><br><br>
+        <button type="submit">Get My Resume</button>
+        <ul class="lang-section">
+          <li>
+            Languages
+            <img src="imgs/Icon-Down.png" alt="#" />
+          </li>
+        </ul>
+        <ul class="lang-prog">
+          <li><img src="imgs/JS.png" alt="#" />Javascript</li>
+          <li><img src="imgs/HTML.png" alt="#" />HTML</li>
+          <li><img src="imgs/CSS.png" alt="#" />CSS</li>
+        </ul>
+        <ul class="lang-section hg">
```

Comment:
- [ ] I would suggest you make the spacing between the logo and content to stick with the Figma design and see the content it's different from your color and font.
### your design
![figma-about](https://user-images.githubusercontent.com/81727988/164911656-30757787-0df5-48ec-92c7-6c9b6fd23336.png)
### Figma design
![about-his](https://user-images.githubusercontent.com/81727988/164911666-8303c501-8dfc-48ff-8bd9-008134b56492.png)